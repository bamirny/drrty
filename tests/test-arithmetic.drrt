def void arithmetic( int x, int y){
	print(x + y);
	print(x-y);
	print(x*y);
	print(x/y);
	printb(x == y);
	printb(x == x);
	printb(x!=y);
	printb(x!=x);
	printb(x>y);
	printb(x>=y);
	printb(x<y);
	printb(x<=y);
}

def int main(){
	arithmetic(100,10);
	arithmetic( 50,25);
	return 0;
	

}	