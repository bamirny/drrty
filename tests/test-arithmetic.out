110<br>
90<br>
1000<br>
10<br>
0<br>
1<br>
1<br>
0<br>
1<br>
1<br>
0<br>
0<br>
75<br>
25<br>
1250<br>
2<br>
0<br>
1<br>
1<br>
0<br>
1<br>
1<br>
0<br>
0<br>
